libs/sdk-ui-dashboard/src/presentation/componentDefinition/types.ts

```diff
@@ -102,68 +145,114 @@ export type CreatableByDragComponent = DraggableComponent & {
         /**
          * Component used to render the item in the left drawer menu used to create a new instance of this component on the dashboard
          */
-        DrawerItemComponent: ComponentType;
+        CreatePanelItemComponent: CustomCreatePanelItemComponent;
```

Comment:
Just one "Hnidopich" item :-). CreatePanelItemComponent this name is so generic or not descriptive what this component is for. 

What about change this name to PanelItemComponentTemplate or PanelItemCreator or PanelItemPlaceholder ?

Reply:
Renamed to CreatePanelListItemComponent, I wanted to avoid the placeholder term as that means something different.